glib/gobject-sys/src/manual.rs

```diff
@@ -0,0 +1,10 @@
+use glib_sys::GType;
+
+// `g_param_spec_types` extern binding generated by build.rs
+include!(concat!(env!("OUT_DIR"), "/param_spec.rs"));
```

Comment:
It would be great if we could avoid such things because that makes the crate unusable with meson's cargo subproject support for the time being.

Reply:
Yeah, I agree, I'm not too happy with the current solution either. Ideally we could skip the code generation part and have the attribute read the link name from an env variable set by the build.rs, or something similar. However, it seems that derive attributes can't read constants [yet](https://github.com/rust-lang/rust/issues/52393).

```
#[cfg_attr(target_os = "windows", link(name = env!("...")))]
extern "C" {
    pub static g_param_spec_types: *const GType;
}
```